content/blog/pulumi-auth0/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Credijusto Manages Authentication with Auth0 and Pulumi"
+date: 2020-09-22
+draft: true
+meta_desc: " Simplifying authentication to APIs with the Pulumi Auth0 provider"
+meta_image: credijusto.png
+authors:
+    - fernando-carletti
+tags:
+    - auth0
+    - Credijusto
+---
+
+*Guest author Lead Devops Engineer Fernando Carletti, writes about using the Pulumi Auth0 provider to manage resources at Credijusto.*
+
+Auth0 allows you to simplify your authentication process. The [Auth0 Provider]({{< relref "/docs/reference/pkg/auth0" >}}) allows you to manage the Auth0 resources, managing Applications, Databases, Social Connections, APIs, and other resources. Here at [Credijusto](https://credijusto.com) we use it manage authentication from the front-end through all the APIs that serve that request, leveraging the complexity of the authentication to Auth0.
+
+For this article, we will start a new Pulumi project in a fresh Auth0 account and fully configure it for a backend and a single page application and set up a connection to Github which allows you apps to authenticate with it using OAuth.
+
+<!--more-->
+
+## Setup the project
+
+Create a new Pulumi program:
+
+```bash
+$ pulumi new typescript
+```
+
+Install the Auth0 provider SDK:
+
+```bash
+$ npm add @pulumi/mysql
+```
+
+Create the Auth0 resources in index.ts as shown in the example below.
```

Comment:
Should there be an example below this?

Reply:
He's referring to the index.ts that was just created. I changed it a little to clarify it.